packages/url-loader/tests/plugins/remove-background.spec.ts

```diff
@@ -25,17 +25,5 @@ describe("Plugins", () => {
 
       expect(cldImage.toURL()).toContain(`e_background_removal`);
     });
-
-    it("should not remove the background", () => {
```

Comment:
curious whats wrong with tests like these?

Reply:
This test no longer works because the new plugin logic enforces that `RemoveBackgroundPlugin` would never be invoked unless one of the props it owns is present.

Ideally since the plugins are an implementation detail, these tests would just go through `getCloudinaryUrl` directly so this wouldn't be possible at all, but with the new design, we know that unless one of those props is present, it's handled centrally instead of per plugin that the props aren't added. If they were added anyways, essentially all the other tests would fail.